web/auth.go

```diff
@@ -12,7 +12,7 @@ import (
 	jwt "github.com/dgrijalva/jwt-go"
 	"github.com/dgrijalva/jwt-go/request"
 	"github.com/gorilla/sessions"
-	"github.com/pressly/lg"
+	"github.com/src-d/lookout/util/ctxlog"
```

Comment:
Since `ctxlog.RequestLogger` is going to be used in `web` I'd move it under `web` dir.
Is there any plan to use it elsewhere?

Reply:
I would keep it here because the middleware belongs to `ctxlog` :)
So most probably it's about personal preferences.
Let's hear one more opinion from @carlosms and move/keep depends on it.

Reply:
To me it feels more like something makes use of `ctxlog`. But I don't see any problem with it in the `ctxlog` package, even if it's not my initial intuition.